genie_registry/maf.py

```diff
@@ -13,7 +13,7 @@
 
 def _check_tsa1_tsa2(df):
```

Comment:
NIT: Could you change the title of the function  (and the test function name) to be something more informative like `_check_allele_col_relationships`?

Reply:
Updated function name to `_check_allele_col_validity`